app/src/main/java/org/schabi/newpipe/player/helper/PlayerHelper.java

```diff
@@ -51,22 +51,17 @@
 import java.lang.annotation.Retention;
 import java.text.DecimalFormat;
 import java.text.NumberFormat;
-import java.util.ArrayList;
 import java.util.Collections;
-import java.util.Formatter;
-import java.util.HashSet;
 import java.util.List;
-import java.util.Locale;
 import java.util.Objects;
 import java.util.Set;
-import java.util.concurrent.TimeUnit;
+import java.util.logging.Logger;
 
 public final class PlayerHelper {
-    private static final StringBuilder STRING_BUILDER = new StringBuilder();
-    private static final Formatter STRING_FORMATTER =
-            new Formatter(STRING_BUILDER, Locale.getDefault());
     private static final NumberFormat SPEED_FORMATTER = new DecimalFormat("0.##x");
     private static final NumberFormat PITCH_FORMATTER = new DecimalFormat("##%");
+    private static final long MEGABYTE = 1024 * 1024L;
+    private static final Logger LOGGER = Logger.getLogger(getClass().getName());
```

Comment:
On Android you don't use `Logger`s, but rather `Log.d/i/w/e(TAG, message[, exception])`